singIn.js

```diff
@@ -0,0 +1,36 @@
+$(document).ready(function() {
+  var $SingInButtom = $('#singInButton');
+  var $SingInModal = $('#singInModal');
+  var $SingUpButton = $('#singUpButton');
+  var $SingUpModal = $('#singUpModal');
+  var $SingInSingUp = $('#singInSingUp');
```

Comment:
needs a noun to know what does it represent :) like `button` or `section` or whatever, at the end